LISA/Executor.cpp

```diff
@@ -343,39 +348,31 @@ uint32_t Executor::GetProgress(const std::string& handle, std::uint32_t& progres
     }
 }
 
-bool Executor::getStorageParamsValid(const std::string& type,
-        const std::string& id,
-        const std::string& version) const
-{
-    // In Stage 1 we support only none parameters or all of them
-    return ((!type.empty() && !id.empty() && !version.empty()) || (type.empty() && id.empty() && version.empty()));
-}
-
 uint32_t Executor::GetStorageDetails(const std::string& type,
                            const std::string& id,
                            const std::string& version,
                            Filesystem::StorageDetails& details)
 {
     namespace fs = Filesystem;
-    if(!getStorageParamsValid(type, id, version)) {
-        return ERROR_WRONG_PARAMS;
-    }
+
     try {
-        if(type.empty()) {
+        if(type.empty() && id.empty() && version.empty()) {
```

Comment:
This got more complex than it was before with just type.empty() check. Could you add a comment what is happening here?

Reply:
Added extra comments